### PR TITLE
chore(DashboardHeader): align DashboardHeader with core

### LINKED ISSUE #7429: [Dashboard wrapper] - [Update to Match core demos]
**Is this a new component or an extension of an existing one?**
What is the existing component, if any?
Dashboard wrapper

**Describe the feature**
A clear and concise description of the new feature. What is the expected behavior?
Core has updated their fullscreen dashboard demos.  React should update the dashboard wrapper so that our full screen demos match core.

### PR DESCRIPTION
**Pull request by jenny-s51**
**What**: Closes #7429

### CODE REVIEW

**Review by nicolethoen — COMMENTED** [review]
Looks like the left/right padding in the masthead, as well as the image size of the PatternFly logo are both a little larger on the react dashboard wrapper than the core demos. 

Jun-07-2022 13-35-14

Also, should this update include updates to the masthead in the page demos? Those are the only full page demos, i believe, that don't use the dashboard wrapper.

**Review by jenny-s51 — COMMENTED** [review]
Thank you for your feedback @nicolethoen ! I think that gif is showing the difference between the react workspaces, i.e. on main vs on this branch. Did you mean to show the difference between the react and core workspaces?

I've increased the size of the masthead inset padding to match the size of the page inset as shown in the core demo, which might be causing the toolbar item spacing to look different by a few pixels. I've also increased the Brand/logo size to match the exact size in the core demos too.

I checked all the HTML elements and the components/styling all do seem to match core. Looks like the toolbar items are still a few pixels away from their placement in core though... would be awesome to get some opinions/feedback on this styling from the core folks! @mattnolting @mcoker 

Also updating the page demos is a great idea! I opened

**Review by jenny-s51 — COMMENTED** [review]
Also, yes, it looks like we do need to update `Page` with the new DashboardWrapper @nicolethoen ! I've opened and assigned myself to https://github.com/patternfly/patternfly-react/issues/7526! Will plan to update those full screen demos with these changes once they are approved 👍

**Comment by nicolethoen** [comment]
> Did you mean to show the difference between the react and core workspaces?

oh - you're right. I thought I had the core demo up... my bad.

**Review by nicolethoen — APPROVED** [review]

**Review by thatblindgeye — APPROVED** [review]

File: packages/react-core/src/demos/examples/DashboardHeader.js [hunk 1]
```diff
@@ -147,7 +146,7 @@ export default class DashboardHeader extends React.Component {
                 <Button aria-label="Help" variant={ButtonVariant.plain} icon={<QuestionCircleIcon />} />
               </ToolbarItem>
             </ToolbarGroup>
-            <ToolbarItem visibility={{ default: 'hidden', sm: 'visible', md: 'visible', lg: 'hidden' }}>
+            <ToolbarItem visibility={{ default: 'visible', sm: 'visible', md: 'visible', lg: 'hidden', xl: 'hidden' }}>
```
Comment (mcoker) [inline packages/react-core/src/demos/examples/DashboardHeader.js:149]: Untested, but I believe all but `lg` are needed - it should be visible by default, and once it's set at a breakpoint, it persists through all breakpoints by default.

```suggestion
            <ToolbarItem visibility={{ lg: 'hidden' }}>
```

**Review by mcoker — APPROVED** [review]
LGTM!!!

**Pull request merged**